feature-detects/css/remunit.js

```diff
@@ -25,7 +25,7 @@ define(['Modernizr', 'createElement'], function( Modernizr, createElement ) {
     try {
       div.style.fontSize = '3rem';
     }
-    catch( er ) {}
+    catch ( er ) {}
```

Comment:
looks like google aligns pretty closely with our own style. it generates the least amount of style changes from all of the presets (crockford is 7k haha).